cmd/eval-dev-quality/cmd/report.go

```diff
@@ -80,6 +81,28 @@ func (command *Report) Execute(args []string) (err error) {
 		command.logger.Panicf("ERROR: %s", err)
 	}
 
+	// Create a CSV file that holds the models total scores along with the corresponding model costs.
```

Comment:
I guess I would really just create a csv with all the meta information. Not already prepare the required data points for the scatter plot. Basically the JS part gets the data in the most bare bone form and then needs to i.e. match that model `A` has score `1234` (from `evaluation.csv`) and cost `0.5` (from `meta.csv`). Cause then we will have that logic ONLY in the JS part. Right now the Go part already knows that the JS part needs cost vs. score for a scatter plot and needs to prepare the data accordingly.